src/parseTools.js

```diff
@@ -1264,6 +1264,16 @@ function hasExportedFunction(func) {
   return Object.keys(WASM_EXPORTS).includes(_asmjsDemangle(func));
 }
 
+// Returns true if the given function is exported out in the generated JS Module object.
+function hasExportedFunction(func) {
+  return Object.keys(EXPORTED_FUNCTIONS).includes(func);
+}
```

Comment:
I reverted my change that removed `hasExportedFunction` in #14024.  Since its based on `WASM_EXPORTS` and not `EXPORTED_FUNCTIONS` I think it does what you want here.